app/views/pages/tutorials/forms/examples.liquid

```diff
@@ -9,7 +9,7 @@ searchable: true
 
 This is a collection of Form Configuration examples from other topics in our documentation.
 
-* [Building a Contact Form with Customization](/tutorials/customizations/building-contact-form-with-customization#step-2-define-contact-form-–-form-object)
+* [Building a Contact Form with Customization](/tutorials/models/building-contact-form-with-customization#step-2-define-contact-form-–-form-object)
```

Comment:
heh, its impossible to detect those kinds of errors, because our system just loads different page under this slug if it doesnt exist...